src/Compilers/VisualBasic/BasicAnalyzerDriver/VisualBasicDeclarationComputer.vb

```diff
@@ -61,36 +61,52 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     For Each decl In t.Members
                         ComputeDeclarationsCore(model, decl, shouldSkip, getSymbol, builder, newLevel, cancellationToken)
                     Next
-                    builder.Add(GetDeclarationInfo(model, node, getSymbol, cancellationToken))
+                    Dim attributes = GetAttributes(t.EnumStatement.AttributeLists)
```

Comment:
>GetAttributes(t.EnumStatement.AttributeLists) [](start = 37, length = 45)

Is this going to produce duplicate code blocks, it looks like we are handling EnumStatement below?

Reply:
No, we added both cases because it turned out symbol's have the statement as their declaring syntax and some have the block. When we are computing declarations in a node we will hit only one of these code paths (note that for `Case SyntaxKind.EnumBlock`, we invoke `ComputeDeclarationsCore` only for its members.

---
In reply to: [152415594](https://github.com/dotnet/roslyn/pull/23318#discussion_r152415594) [](ancestors = 152415594)

Reply:
Filed [https://github.com/dotnet/roslyn/issues/23347](https://github.com/dotnet/roslyn/issues/23347) to ensure we have clarity here.

---
In reply to: [152420875](https://github.com/dotnet/roslyn/pull/23318#discussion_r152420875) [](ancestors = 152420875,152415594)